coraza.conf-recommended

```diff
@@ -230,6 +230,10 @@ SecAuditLogParts ABIJDEFHZ
 #
 SecAuditLogType Serial
 
+# The format used to write the audit log.
+# Can be one of JSON|Native|OCSF
+SecAuditLogFormat OCSF
```

Comment:
I don't think we want to recommend this by default as this feels like a breaking change.

Reply:
Agreed. We should keep the native as default for now.

Reply:
I've reverted the default back to "Native".